src/web/mjs/connectors/GoComics.mjs

```diff
@@ -0,0 +1,116 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class GoComics extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'gocomics';
+        super.label = 'GoComics';
+        this.tags = ['comic', 'english'];
+        this.url = 'https://www.gocomics.com';
+        this.config = {
+            throttle: {
+                label: 'Throttle Requests [ms]',
+                description: 'Enter the timespan in [ms] to delay consecuitive HTTP requests.\nThe website may ban your IP for too many consecutive requests.',
+                input: 'numeric',
+                min: 0,
+                max: 5000,
+                value: 500
+            }
+        };
+        this.lastChapter = '';
+        this.getPagesCount = 0;
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let comicData = await this.fetchDOM(request, 'nav.content-section-padded-sm ul a');
+        let id = comicData[1].pathname.trim();
+        let titleData;
+        if (new RegExp(/\/\d\d\d\d\/\d?\d\/\d?\d$/).test(uri) == true) {
+            titleData = await this.fetchDOM(request, 'a.gc-blended-link div.media.amu-media-item div.media-body h2');
+        } else {
+            titleData = await this.fetchDOM(request, 'a.gc-blended-link div.media.amu-media-item div.media-body h1');
+        }
+        let title = titleData[0].innerText.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(this.url + '/comics/a-to-z', this.requestOptions);
+        let data = await this.fetchDOM(request, 'a.gc-blended-link.gc-blended-link--primary.col-12.col-sm-6.col-lg-4');
+        return data.map(element => {
+            let title = element.querySelector('div.media-body h4');
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: title.innerText.trim()
+            };
+        });
+    }
+
+    async _getChapters(manga) {
+        let chapter = manga.id;
+        this.lastChapter = manga.id;
+        let request = new Request(this.url + manga.id, this.requestOptions);
+        let startData = await this.fetchDOM(request, 'nav.gc-calendar-nav a.fa-backward');
+        let startDate = this._getDate(startData[0].pathname);
+        let endDate = this._getDate(manga.id);
+        let chapterList = [];
+        let title = manga.id.slice(1, manga.id.search(/\/\d\d\d\d\/\d?\d\/\d?\d$/));
```

Comment:
May simplify regex, use regex101 online editor for testing and verification
e.g. `/.\d{4}.\d{1,2}.\d{1,2}$/`